randovania/games/am2r/json_data/Main Caves.txt

```diff
@@ -1487,6 +1519,8 @@ Extra - minimap_data: [{'x': 55, 'y': 37}, {'x': 55, 'y': 38}, {'x': 56, 'y': 37
               # Go Through Lava
               Gravity Suit
               Damage Boost (Intermediate) and Lava Damage ≥ 50
+          # Shinespark
+          Speed Booster and Shinesparking Tricks (Beginner) and Short Charge (Intermediate) and Disabled Entrance Rando
```

Comment:
Can you add a Shinespark TODO in Gamma nest/gamma -> dock? It's possible to charge a shinespark there while the flitts are slowing you down, but I don't know the rating right now, and repro'ing it is a bit RNG dependant.